pubsub/topic.go

```diff
@@ -584,5 +584,6 @@ func (t *Topic) publishMessageBundle(ctx context.Context, bms []*bundledMessage)
 // encountered while publishing, to prevent messages from being published
 // out of order.
 func (t *Topic) ResumePublish(orderingKey string) {
+	t.initBundler()
```

Comment:
I think I prefer just checking `if t.scheduler != nil` here. While functionally similar, I think `ResumePublish` should not handle bundle initialization since it should normally only be called after `Publish` has been called.

Reply:
Amended.